packages/fxa-payments-server/src/routes/Product/SubscriptionRedirect/index.tsx

```diff
@@ -1,33 +1,44 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { Plan } from '../../../store/types';
 
 import './index.scss';
 
+// Mapping of product IDs to post-subscription redirect URLs
+// TODO: Move / merge in some kind of configuration variable?
+type productRedirectURLsType = { [productID: string]: string };
+const productRedirectURLs: productRedirectURLsType = {
+  '123doneProProduct': 'http://127.0.0.1:8080/',
+  'prod_Ex9Z1q5yVydhyk': 'https://123done-latest.dev.lcip.org/',
+  'prod_FUUNYnlDso7FeB': 'https://123done-stage.dev.lcip.org',
+};
+const defaultProductRedirectURL = 'https://mozilla.org';
+
 export type SubscriptionRedirectProps = {
   plan: Plan,
   navigateToUrl: Function,
 };
 
-// Table of lazy-loaded plan detail components - the keys are plan IDs.
-type availableRedirectsType = {
-  [propName: string]: React.LazyExoticComponent<(props: SubscriptionRedirectProps) => JSX.Element>
-};
-const availableRedirects: availableRedirectsType = {
-  // Examples:
-  '123doneProProduct': React.lazy(() => import('./Redirect123donePro')),
-  // '321doneProProduct': React.lazy(() => import('./Redirect321donePro')),
-  'prod_Ex9Z1q5yVydhyk': React.lazy(() => import('./RedirectDev')),
-};
-const defaultRedirect = React.lazy(() => import('./RedirectDefault'));
-
 export const SubscriptionRedirect = ({
-  plan,
   navigateToUrl,
+  plan: {
+    product_id,
+    plan_name,
+  }
 }: SubscriptionRedirectProps) => {
-  const SubRedirect = plan.product_id in availableRedirects
-    ? availableRedirects[plan.product_id]
-    : defaultRedirect;
-  return <SubRedirect {...{ plan, navigateToUrl }} />;
+  const redirectUrl = productRedirectURLs[product_id]
+    || defaultProductRedirectURL;
```

Comment:
The config.js file can also supply a default, so, as a bonus, we can eliminate branches like this in the code.

Reply:
Well, the default is here but mostly never meant to be used - in local dev or others, it's going to be possible that some product IDs are tinkered with before they end up in config